README.md

```diff
@@ -1,30 +1,30 @@
-## Example Kit Integration
+## Swrve Kit Integration
 
-This repository contains the [Example](https://www.example.com/) integration for the [mParticle Apple SDK](https://github.com/mParticle/mparticle-apple-sdk).
+This repository contains the [SWrve](https://www.swrve.com/) integration for the [mParticle Apple SDK](https://github.com/mParticle/mparticle-apple-sdk).
 
 ### Adding the integration
 
 1. Add the kit dependency to your app's Podfile or Cartfile:
 
     ```
-    pod 'mParticle-Example', '~> 1.2'
+    pod 'mParticle-Swrve', '~> 1.0'
```

Comment:
I think it might be more appropriate to name this `Swrve-mParticle`

Reply:
I think that is probably correct. It will involve a refactor of the structure of the repo - so I will make those changes and push them up for review.